CHANGES.rst

```diff
@@ -6,6 +6,8 @@ assign_wcs
 
 - Update keyword and attribute usage around SkyObject to reflect updated keywords. [#4943]
 
+- Refactor PPS origin to bottom left rather than top left.
```

Comment:
Would be good to be a little more explicit in terms of the instrument mode, i.e. "Refactor PPS origin of NIRSpec MOS shutters to bottom left, rather than top left"